### DISCUSSION TITLE
Move Discussion to Issue

### DISCUSSION THREAD

**Discussion by rpgoldman** [community/community#3869] [Discussions]
It would be great if a user question in Discussions could be moved to be an Issue.  Users who are not confident may feel more comfortable *asking* whether they have found a bug, rather than filing an issue, and as a maintainer, it would be great if there was a way for me to push a button and turn a discussion question into an issue.

If there *is* such a method, I didn't see anything on the UI that was an obvious affordance...

**Comment by alkisg** [comment]
I completely agree. All these should be possible:

* Moving an issue into discussions
* Regretting the move and moving it back into issues!
* Moving a discussion into an issue

A few months ago, converting an issue into a discussion kept the same issue #number and left a redirect behind. I liked that approach much better than the current "cloning into a discussion while making the current issue read-only" one. This one makes it harder to move discussions back to issues.
